unikernel.ml

```diff
@@ -939,14 +939,24 @@ struct
           ~data:(`String (String.escaped err))
           reqd `Bad_request
 
-  let deploy_form store _ (user : User_model.user) reqd =
+  let deploy_form store albatross _ (user : User_model.user) reqd =
     let now = Mirage_ptime.now () in
+    user_unikernels albatross user.name >>= fun unikernels ->
+    user_volumes albatross user.name >>= fun blocks ->
     generate_csrf_token store user now reqd >>= function
     | Ok csrf ->
+        let user_policy =
+          match Albatross.policy albatross ~domain:user.name with
+          | Ok p -> (
+              match p with Some p -> p | None -> Albatross.empty_policy)
+          | Error _ -> Albatross.empty_policy
```

Comment:
Hmm, why I understand that a default is good, the empty_policy leads to an error later -- since it won't allow any unikernel deployments.

So, I was wondering whether we should directly error out here instead!? And inform the user "you're not allowed to create any unikernels" / "couldn't retrieve resource policy from albatross".